BasicSample/ProjectSettings/ProjectSettings.asset

```diff
@@ -12,8 +12,8 @@ PlayerSettings:
   targetDevice: 2
   useOnDemandResources: 0
   accelerometerFrequency: 60
-  companyName: DefaultCompany
-  productName: OpenXR-Unity-MixedReality-Samples
+  companyName: Microsoft
```

Comment:
Let's revert this to DefaultComany.  Sample code shouldn't encode real company name.

Reply:
I feel like it should represent the author of the sample and can be changed if a dev starts to base their app on the sample code (they'll want to update the `productName` I'm sure, so they can update `companyName` at the same time). Keaton previously mentioned at https://github.com/microsoft/OpenXR-Unity-MixedReality-Samples/pull/5#discussion_r605997932:

> I looked at some related Microsoft samples, and they also had "Microsoft" set in this companyName field. I'll update this here too.

It'd also help if people are digging into folders trying to find the log file for this value to be more than `DefaultCompany`.

Some examples I just found:

https://github.com/Azure/azure-spatial-anchors-samples/blob/08e00c61d419e2e322be883f9adf3ee8bf0eb0ed/Unity/ProjectSettings/ProjectSettings.asset#L15

https://github.com/Azure/azure-object-anchors/blob/d7e6d9d015dac7ee533ca96b6c81ca29cfec839d/quickstarts/apps/unity/basic/ProjectSettings/ProjectSettings.asset#L15

https://github.com/microsoft/HolographicAcademy/blob/4d750ab8018874fa2fe08a7fcf8ae4009268fbb0/Completed/ModelExplorer/ProjectSettings/ProjectSettings.asset#L14

Some use the project name as the `companyName`, but I'm not sure that's properly representative:

https://github.com/microsoft/MixedReality-WorldLockingTools-Samples/blob/ad5feba28c5557554c52ea46ce80879c1eac1663/Advanced/QRSpacePins/ProjectSettings/ProjectSettings.asset#L15

https://github.com/microsoft/HolographicAcademy/blob/712d659b310f87dfa14cad9a7680f3a3f4af4b02/Origami/ProjectSettings/ProjectSettings.asset#L13

Reply:
discussed in scrum, let's change it to "MicrosoftSample"

Reply:
Updated in the latest commits 👍